@navikt/core/react/src/overlays/floating-menu/parts/FocusScope.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from "react";
+import { forwardRef, useEffect, useState } from "react";
+import { Slot } from "../../../slot/Slot";
+import { useCallbackRef, useMergeRefs } from "../../../util/hooks";
+
+const AUTOFOCUS_ON_MOUNT = "focusScope.autoFocusOnMount";
+const AUTOFOCUS_ON_UNMOUNT = "focusScope.autoFocusOnUnmount";
+const EVENT_OPTIONS = { bubbles: false, cancelable: true };
+
+interface FocusScopeProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * Event handler called when auto-focusing on mount.
```

Comment:
```suggestion
   * Event handler called on mount, unless the component already has focus. Used for auto-focusing.
```
I feel this would be more precise, but correct me if I'm wrong. A little unsure about the "unless" part. Not sure if it's actually possible, or if it's just to prevent the callback from being called twice?